### DISCUSSION TITLE
Can't turn off follow mode

### DISCUSSION THREAD

**Discussion by sztomi** [folke/trouble.nvim#597] [Q&A] [answered]
I'm pretty sure I'm holding this wrong, but I tried to disable follow mode, but couldn't. This is my config:

```lua
    {
        "folke/trouble.nvim",
        cmd = "Trouble",
        event = "VeryLazy",
        opts = {
            follow = false,
        },
    }
```

Any tips?

**Comment by pchar2** [comment, marked as answer]
> The follow option means that the cursor in the trouble window will follow where you are in the buffer.

https://github.com/folke/trouble.nvim/issues/430#issuecomment-2111716023

Just in case you thought (as did I) it was the other way around.

**Reply by sztomi** [reply]
Thanks! This was indeed the case for me as well.
